src/main/java/frc/trigon/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -0,0 +1,172 @@
+package frc.trigon.robot.subsystems.elevator;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.signals.*;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import org.trigon.hardware.RobotHardwareStats;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+import org.trigon.hardware.simulation.ElevatorSimulation;
+import org.trigon.utilities.mechanisms.ElevatorMechanism2d;
+
+public class ElevatorConstants {
+    private static final int
+            MASTER_MOTOR_ID = 13,
+            FOLLOWER_MOTOR_ID = 14;
+    private static final String
+            MASTER_MOTOR_NAME = "ElevatorMasterMotor",
+            FOLLOWER_MOTOR_NAME = "ElevatorFollowerMotor";
+    static final TalonFXMotor
+            MASTER_MOTOR = new TalonFXMotor(MASTER_MOTOR_ID, MASTER_MOTOR_NAME),
+            FOLLOWER_MOTOR = new TalonFXMotor(FOLLOWER_MOTOR_ID, FOLLOWER_MOTOR_NAME);
+
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
+    private static final InvertedValue
+            MASTER_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive,
+            FOLLOWER_MOTOR_INVERTED_VALUE = InvertedValue.Clockwise_Positive;
+    private static final double GEAR_RATIO = 20;
+    private static final boolean FOLLOWER_MOTOR_OPPOSES_MASTER = false;
+    private static final double
+            P = RobotHardwareStats.isSimulation() ? 110 : 50,
+            I = RobotHardwareStats.isSimulation() ? 0 : 0,
+            D = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KS = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KV = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KG = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KA = RobotHardwareStats.isSimulation() ? 0 : 0;
+    private static final GravityTypeValue GRAVITY_TYPE_VALUE = GravityTypeValue.Elevator_Static;
+    private static final StaticFeedforwardSignValue STATIC_FEEDFORWARD_SIGN_VALUE = StaticFeedforwardSignValue.UseClosedLoopSign;
+    private static final ReverseLimitSourceValue REVERSE_LIMIT_SOURCE_VALUE = ReverseLimitSourceValue.LimitSwitchPin;
+    private static final ForwardLimitSourceValue FORWARD_LIMIT_SOURCE_VALUE = ForwardLimitSourceValue.LimitSwitchPin;
+    private static final ReverseLimitTypeValue REVERSE_LIMIT_TYPE_VALUE = ReverseLimitTypeValue.NormallyOpen;
+    private static final ForwardLimitTypeValue FORWARD_LIMIT_TYPE_VALUE = ForwardLimitTypeValue.NormallyOpen;
+    private static final double
+            REVERSE_LIMIT_SWITCH_RESET_POSITION = 2,
+            FORWARD_LIMIT_SWITCH_RESET_POSITION = 0.1;
```

Comment:
1. reverse should always be less than forward 
2. we won't want a forward rest 
3. the reverse reset should be to 0